homeassistant/components/zha/manifest.json

```diff
@@ -15,6 +15,12 @@
     "zigpy-zigate==0.7.3",
     "zigpy-znp==0.5.3"
   ],
+  "usb": [
+   {"vid":"10C4","pid":"EA60"},
+   {"vid":"1CF1","pid":"0030"},
+   {"vid":"1A86","pid":"7523"},
+   {"vid":"10C4","pid":"8A2A"}
```

Comment:
I think it would be extremely helpful to have some information about what these devices are. Looking at this list, I have no idea if I can extend this... Even though, I have quite a collection of Zigbee capable USB devices.

Reply:
I'll add a description field once `zwave_js` is merged and back populate them all.

Reply:
https://github.com/bdraco/home-assistant/commit/4d30d1f40d04d57db6182dec118e2f349dc4c028

Reply:
https://github.com/home-assistant/core/pull/54986